src/OnlineSales/Tasks/SyncIPDetailsTask.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the samples root for full license information.
 // </copyright>
 
+using Microsoft.EntityFrameworkCore;
```

Comment:
@WASandaruwan this class is not related with change log logic. Please remove from this PR to keep focus on change log only.